cli/central-client/src/main/java/org/ballerinalang/central/client/CentralAPIClient.java

```diff
@@ -940,6 +943,80 @@ public PackageResolutionResponse resolveDependencies(PackageResolutionRequest re
         }
     }
 
+    /**
+     * Resolve Tools from central.
+     * @param request            The tool resolution request.
+     * @param supportedPlatform  The supported jBallerina backend platform.
+     * @param ballerinaVersion   The ballerina distribution version.
+     *
+     * @throws CentralClientException Central Client exception.
+     */
+    public ToolResolutionCentralResponse resolveToolDependencies(
+            ToolResolutionCentralRequest request, String supportedPlatform, String ballerinaVersion)
+            throws CentralClientException {
+        String url = this.baseUrl + "/" + TOOLS + "/" + RESOLVE_DEPENDENCIES;
```

Comment:
What about using a constant for "/"

Reply:
Fixed with 64f55075148e69e3b5992080ba8c3b817d5cbc61